### ISSUE TITLE
Active delay/trigger is reset when adding/removing a node to its group

### ISSUE THREAD

**Issue by bonanitech** [node-red/node-red#2835]
### What are the steps to reproduce?
Using the code below (for example):

- Hit the inject node to start the delay/trigger node
- Add a new node to the group and Deploy

```json
[{"id":"246be425.21818c","type":"group","z":"1f8f3196.f58a2e","style":{"stroke":"#2b2b2b","stroke-opacity":"1","fill":"none","fill-opacity":"1","label":true,"label-position":"nw","color":"#a4a4a4"},"nodes":["10492191.a8550e","12660740.3c6799","6f5a4b25.488f74","41ae0b43.1d83b4"],"x":324,"y":79,"w":532,"h":142},{"id":"10492191.a8550e","type":"inject","z":"1f8f3196.f58a2e","g":"246be425.21818c","name":"","props":[],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","x":420,"y":150,"wires":[["12660740.3c6799","6f5a4b25.488f74"]]},{"id":"12660740.3c6799","type":"delay","z":"1f8f3196.f58a2e","g":"246be425.21818c","name":"","pauseType":"delay","timeout":"15","timeoutUnits":"minutes","rate":"1","nbRateUnits":"1","rateUnits":"second","randomFirst":"1","randomLast":"5","randomUnits":"seconds","drop":false,"x":600,"y":120,"wires":[["41ae0b43.1d83b4"]]},{"id":"6f5a4b25.488f74","type":"trigger","z":"1f8f3196.f58a2e","g":"246be425.21818c","name":"","op1":"1","op2":"0","op1type":"str","op2type":"str","duration":"15","extend":false,"overrideDelay":false,"units":"min","reset":"","bytopic":"all","topic":"topic","outputs":1,"x":600,"y":180,"wires":[["41ae0b43.1d83b4"]]},{"id":"41ae0b43.1d83b4","type":"debug","z":"1f8f3196.f58a2e","g":"246be425.21818c","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","statusVal":"","statusType":"auto","x":760,"y":150,"wires":[]}]
```

### What happens?
The delay/trigger node inside the group is reset, even with deploy set to "Modified Nodes".

Removing a node from the group has the same effect.

### What do you expect to happen?
Delay/trigger timer to continue running.

### Please tell us about your environment:

- [ ] Node-RED version: 1.2.7
- [ ] Node.js version:
- [ ] npm version:
- [ ] Platform/OS:
- [ ] Browser:

**Comment by hardillb** [comment]
What sort of deploy did you do?

**Comment by bonanitech** [comment]
I forgot to mention, moving the group around doesn't reset the delay/trigger timer.

**Comment by bonanitech** [comment]
Should I close this issue now?

**Comment by knolleary** [comment]
It'll get closed automatically when the fix is merged to master as part of 1.3 release. But you're welcome to close it if you'd like.
